pkg/resources/fluentd/service.go

```diff
@@ -91,6 +91,10 @@ func (r *Reconciler) monitorServiceMetrics() (runtime.Object, reconciler.Desired
 				objectMetadata.Labels[k] = v
 			}
 		}
+		metricScrapePath := r.Logging.Spec.FluentdSpec.Metrics.Path
+		if r.Logging.Spec.FluentdSpec.Workers > 1 {
+			metricScrapePath = "/aggregated_metrics"
```

Comment:
If you override the metrics path here, users won't be able to customize the path (if they are using multiple workers). Instead, the default value should be set depending on the number of workers here: https://github.com/banzaicloud/logging-operator/blob/6b16d314e8f0da31ee292ab930150467af11bdd3/pkg/sdk/logging/api/v1beta1/logging_types.go#L181